src/middleware/finalErrorHandler.js

```diff
@@ -0,0 +1,6 @@
+const { messages: errMessages } = require('../constants/errors')
+
+module.exports = (err, req, res) => {
```

Comment:
from the [express website (see the error handler section)](https://expressjs.com/en/guide/using-middleware.html):
>Error-handling middleware always takes four arguments. You must provide four arguments to identify it as an error-handling middleware function. Even if you don’t need to use the next object, you must specify it to maintain the signature. Otherwise, the next object will be interpreted as regular middleware and will fail to handle errors.

Other wise it assumes it's normal middleware so `(err, req, res)` becomes `(req, res, next)` and you get a `res.boom` doesn't work (which is how I discovered all this)